examples/pub-sub-occupancy/react/src/App.tsx

```diff
@@ -5,9 +5,13 @@ import { useEffect, useState } from 'react';
 import { FaEye } from 'react-icons/fa';
 import './styles/styles.css';
 
+const urlParams = new URLSearchParams(typeof window !== 'undefined' ? window.location.search : '');
+const channelName = urlParams.get('name') || 'pub-sub-occupancy';
```

Comment:
Does this work okay? Usually you'd access this sort of stuff inside a `useEffect` inside the component that runs on mount (i.e. with a dependency array of []) - that's the Reacty way. But given the simplicity and clientside nature of the app, might be alright.

Reply:
It does indeed work.